code/game/turfs/simulated/floor/reinf_floor.dm

```diff
@@ -25,18 +25,21 @@
 		..()
 	return //unplateable
 
-/turf/open/floor/engine/attackby(obj/item/C, mob/user, params)
-	if(!C || !user)
+/turf/open/floor/engine/wrench_act(mob/living/user, obj/item/wrench/W)
+	if(!W || !user)
```

Comment:
Is this check needed? None of the other tool acts that currently exist check for these, despite doing very similar things in most cases

Reply:
Probably not